jdiagram/build.gradle

```diff
@@ -0,0 +1,29 @@
+plugins{
+    id 'library.deps-conventions'
+    id 'library.java-conventions'
+
+    // The following plugin is used to build the OSGI bundle
+    id 'biz.aQute.bnd.builder' version '6.4.0'
+}
+
+dependencies {
+    //module dependencies
+    implementation project(':wrims-ide')
+    implementation project(':dwr-hecdssvue')
+
+    //additional dependencies
+    implementation libs.hec.monolith
```

Comment:
Same comment as: https://github.com/CentralValleyModeling/wrims/pull/138/files#r1823248943

Reply:
Same reply: We will verify this consolidation with Hao and merge the hec-dependencies module into third-party after the installer is building successfully. Tracking that under this issue: https://github.com/CentralValleyModeling/wrims/issues/152